src/main/java/com/jcabi/github/Issues.java

```diff
@@ -85,4 +87,116 @@ Issue create(
     Iterable<Issue> iterate(@NotNull(message = "map of params can't be NULL")
         Map<String, String> params);
 
+    /**
+     * Search for issues within the given repository.
+     *
+     * @param sort The sort field
+     * @param direction The sort direction
+     * @param qualifiers The search qualifier
+     * @return Issues
+     * @throws java.io.IOException If there is any I/O problem
+     * @since 0.22.0
+     * @see <a href="https://developer.github.com/v3/issues/#list-issues-for-a-repository">List issues for a repository</a>
+     */
+    @NotNull(message = "Iterable of issues is never NULL")
+    Iterable<Issue> search(
+        @NotNull(message = "Sort field can't be NULL") Sort sort,
+        @NotNull(message = "Sort direction can't be NULL")
+        Search.Order direction,
+        @NotNull(message = "Search qualifiers can't be NULL")
+        EnumMap<Qualifier, String> qualifiers)
```

Comment:
I'm basically just cloning the interface used by https://github.com/jcabi/jcabi-github/blob/cb7ac2fcdfca0ccbfaefbc9109c6e4ec9f5c60fd/src/main/java/com/jcabi/github/RtSearch.java#L119
I have no opinion/suggestion as to whether `EnumMap` is appropriate.